openshift-kube-apiserver/authentication/oauth/bootstrapauthenticator.go

```diff
@@ -17,16 +18,18 @@ import (
 const ClusterAdminGroup = "system:cluster-admins"
 
 type bootstrapAuthenticator struct {
-	tokens    oauthclient.OAuthAccessTokenInterface
-	getter    bootstrap.BootstrapUserDataGetter
-	validator OAuthTokenValidator
+	tokens            oauthclient.OAuthAccessTokenInterface
```

Comment:
squash into code introducing commit

Reply:
Done. Squashed with `UPSTREAM: <carry>: collapse closer to kube authentication stack`.